File: PYTHON/Colleen.py
# Comment out

def quine():
    s = "# Comment out%c%cdef quine():%c    s = %c%s%c%c    print(s%%(10,10,10,34,s,34,10,34,34,10,10,10,10,10,10,34,34,10), end=%c%c)%c%cdef main():%c    # Comment in%c    quine()%c%cif __name__ == %c__main__%c:%c    main()"
    print(s%(10,10,10,34,s,34,10,34,34,10,10,10,10,10,10,34,34,10), end="")

def main():
    # Comment in
    quine()

if __name__ == "__main__":
    main()

File: C/Colleen.c
#include <stdio.h>

/*
  Comment out
*/

void quine() {

	char *code = "#include <stdio.h>%c%c/*%c  Comment out%c*/%c%cvoid quine() {%c%c	char *code = %c%s%c;%c	printf(code, 10, 10, 10, 10, 10, 10, 10, 10, 34, code, 34, 10, 10, 10, 10, 10, 10,10,10,10,10,10, 10, 10, 10);%c}%c%cint main(void) {%c%c	/*%c	  Comment in%c	*/%c%c	quine();%c%c	return (0);%c}";
	printf(code, 10, 10, 10, 10, 10, 10, 10, 10, 34, code, 34, 10, 10, 10, 10, 10, 10,10,10,10,10,10, 10, 10, 10);
}

int main(void) {

	/*
	  Comment in
	*/

	quine();

	return (0);
}